packages/grpc-native-core/index.d.ts

```diff
@@ -258,17 +260,21 @@ declare module "grpc" {
     responseDeserialize: deserialize;
   }
 
-  type handleCall = handleUnaryCall | handleClientStreamingCall | handleServerStreamingCall | handleBidiStreamingCall;
+  type handleCall =
+    handleUnaryCall<any> |
+    handleClientStreamingCall<any> |
+    handleServerStreamingCall<any> |
+    handleBidiStreamingCall<any, any>;
 
   /**
    * User-provided method to handle unary requests on a server
    */
-  type handleUnaryCall = (call: ServerUnaryCall, callback: sendUnaryData) => void;
+  type handleUnaryCall<T> = (call: ServerUnaryCall<T>, callback: sendUnaryData<T>) => void;
```

Comment:
Later in the file, `ServerUnaryCall` is defined in terms of the request type, and `sendUnaryData` is defined in terms of the response type, but you are using just one type parameter here for both. This is incorrect.

Reply:
Ok, I did not dig deep enough into the API, yes, so this should be revised.